### PR TITLE
[service.subtitles.legendasdivx] v1.0.5 - Code fix. Login and Regex.

### PR DESCRIPTION
**Pull request by highlandr**
Fix Login
Fix Regex

### Checklist:

- [x ] My code follows the [add-on rules](http://kodi.wiki/view/Add-on_rules) and [piracy stance](http://kodi.wiki/view/Official:Forum_rules#Piracy_Policy) of this project. 
- [ x] I have read the [CONTRIBUTING](https://github.com/xbmc/repo-scripts/blob/master/CONTRIBUTING.md) document
- [ x] Each add-on submission should be a single commit with using the following style: [script.foo.bar] 1.0.0

Additional information :
- Submitting your add-on to this specific branch makes it available to any Kodi version equal or higher than the branch name with the applicable Kodi dependencies limits.
- [add-on development](http://kodi.wiki/view/Add-on_development) wiki page.
- Kodi [pydocs](http://kodi.wiki/view/PyDocs) provide information about the Python API
- [PEP8](https://www.python.org/dev/peps/pep-0008/) codingstyle which is considered best practise but not mandatory.
- This add-on repository has automated code guideline check which could help you improve your coding. You can find the results of these check at [Codacy](https://www.codacy.com/app/Kodi/repo-scripts/dashboard). You can create your own account as well to continuously monitor your python coding before submitting to repo.
- Development questions can be asked in the [add-on development](http://forum.kodi.tv/forumdisplay.php?fid=26) section on the Kodi forum.

### CODE REVIEW

**Comment by highlandr** [comment]
@enen92 Can you check this out? Been a while since I didn't add new code to this plugin :) at the moment is broken and needs this update. Thanks. Cheers!

File: service.subtitles.legendasdivx/service.py [hunk 1]
```diff
@@ -293,293 +308,288 @@ def append_subtitle(item):
...
\ No newline at end of file
+    def Search(item):
+        """Called when searching for subtitles from KODI."""
+        #### Do what's needed to get the list of subtitles from service site
+        #### use item["some_property"] that was set earlier
+        #### once done, set xbmcgui.ListItem() below and pass it to xbmcplugin.addDirectoryItem()
+        #### CHECKING FOR ANYTHING IN THE USERNAME AND PASSWORD, IF NULL IT STOPS THE SCRIPT WITH A WARNING
+        username = _addon.getSetting( 'LDuser' )
+        password = _addon.getSetting( 'LDpass' )
+        if username == '' or password == '':
```
Comment (enen92) [inline service.subtitles.legendasdivx/service.py:319]: You could make this a bit more pythonic by doing something like:

`if not username or not password:`

File: service.subtitles.legendasdivx/service.py [hunk 2]
```diff
@@ -293,293 +308,288 @@ def append_subtitle(item):
...
+            xbmcplugin.endOfDirectory(int(sys.argv[1]))
+            if username == '' and password != '': xbmc.executebuiltin(('Notification(%s,%s,%d)' % (_scriptname , _language(32016).encode('utf-8'),5000)))
```
Comment (enen92) [inline service.subtitles.legendasdivx/service.py:321]: Can you please avoid the built-in function here? We have access to this on the python API:
xbmcgui.Dialog().notification()

**Review by enen92 — CHANGES_REQUESTED** [review]
All good execpt for the notification usage as bultins. We should avoid that as much as possible if the same functions are available on the python interface.
It is required to avoid the sys.path.append() usage and use relative imports instead:

`from resources.lib import x`

However, as this was not introduced in the scope of this PR you can do it in a later update.

File: service.subtitles.legendasdivx/service.py (hunk 1 above)
Comment (highlandr) [reply service.subtitles.legendasdivx/service.py]: Done @enen92

File: service.subtitles.legendasdivx/service.py (hunk 2 above)
Comment (highlandr) [reply service.subtitles.legendasdivx/service.py]: Done.

**Review by enen92 — APPROVED** [review]

**Pull request merged**